README.md

```diff
@@ -996,16 +996,23 @@ should also work if `oomph-lib` has already been configured, using a build direc
 
 **WORK IN PROGRESS**
 
-## Build options
+---
+Start MH edits
+
+---
+
+## Customising the build options
 
 ### Configuring the build via the specification of CMake cache variables
 
-You can customise your build by passing flags of the form `-D<FLAG>=<value>` to `cmake` during the configuration/generation ** __Puneet:__ why "generation"? *** step. For instance, `oomph-lib` allows range checking (useful for debugging but costly in terms of runtime) via a C++ macro  `RANGE_CHECKING`; if this macro is defined, `oomph-lib` will check that indices passed to vectors (and similar containers) are within the legal range. This requires the `oomph-lib` source code to be compiled with appropriate compiler flags, as shown here
+You can customise your build by passing flags of the form `-D<FLAG>=<value>` to `CMake` during the configuration step. For instance, `oomph-lib` allows range checking (useful for debugging but costly in terms of run time) via the specification of a C++ macro  `RANGE_CHECKING`; if this macro is defined, `oomph-lib` will check that indices passed to vectors (and similar containers) are within the legal range. This requires the `oomph-lib` source code to be compiled with appropriate compiler flags, e.g.
 ```bash
 g++ -DRANGE_CHECKING my_code.cc
 ```
-To ensure that the relevant compiler flags are specified at compile time, we employ so-called CMake cache variables which have to be specified at the configure stage. For instance, to enable range checking for all source code in the library, set the cache variable `OOMPH_ENABLE_RANGE_CHECKING` to `ON` when configuring the project:
-***__Puneet__: Please check: No need to specify the flag during the build and install stages, right? ***
+if gcc is used as the compiler.
```

Comment:
"gcc" -> "GCC" if referring to the compiler collection or "gcc" -> "g++" if referring to the GCC C++ compiler